modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/js/components/baseComponents/BaseAPIEndpointFields.tsx

```diff
@@ -355,56 +446,57 @@ export default function BaseAPIEndpointFields({
 						/>
 					</ClayInput.GroupItem>
 
-					{selectedRetrieveType?.value === 'singleElement' && (
-						<>
-							<ClayInput.GroupItem
-								prepend
-								shrink
-								style={{marginLeft: 0}}
-							>
-								<ClayInput.GroupText>/</ClayInput.GroupText>
-							</ClayInput.GroupItem>
-
-							<ClayInput.GroupItem
-								append
-								className={classNames({
-									'has-error': displayError.parameter,
-								})}
-							>
-								<ClayInput
-									aria-label={endpointParameterLabel}
-									id="endpointParameterField"
-									onBlur={() =>
-										setData((previousData) => ({
-											...previousData,
-											parameter: stringBetweenCurlyBraces(
-												removeLeadingForwardSlash(
-													previousData.parameter!
-												)
-											),
-										}))
-									}
-									onChange={({target: {value}}) =>
-										setData((previousData) => ({
-											...previousData,
-											parameter: makeURLPathParameterString(
-												value
-											),
-										}))
-									}
-									placeholder={endpointParameterLabel}
-									type="text"
-									value={
-										data.parameter
-											? removeLeadingForwardSlash(
-													data.parameter
-											  )
-											: ''
-									}
-								/>
-							</ClayInput.GroupItem>
-						</>
-					)}
+					{selectedHttpMethod?.value === 'get' &&
```

Comment:
these `'get'`,` 'retrieveType'`, `'single-element'`... that I see along the code should be constants.